data/sql/updates/pending_db_world/rev_1637473250058502604.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1637473250058502604');
+
+DROP TABLE IF EXISTS `playercreateinfo_cast_spell`;
+CREATE TABLE IF NOT EXISTS `playercreateinfo_cast_spell` (
+    `raceMask` INT(10) UNSIGNED NOT NULL DEFAULT '0',
```

Comment:
we don't specify length anymore, I can't find where it's documented though @Kitzunu

Reply:
https://www.azerothcore.org/wiki/sql-standards#tables-and-columns